### PR TITLE
CNV2525 - extended Telemetry metrics to include VMs and VMIs

### PR DESCRIPTION
**Pull request by aburdenthehand**
Added conditional information that includes VM and VMI count in the Telemetry 'what info is collected' list

### CODE REVIEW

File: modules/telemetry-what-information-is-collected.adoc [hunk 1]
```diff
@@ -18,6 +22,10 @@ Primary information collected includes:
 * Number of members in the etcd cluster and number of objects currently stored in the etcd cluster
 * Number of CPU cores and RAM used per machine type - infra or master
 * Number of CPU cores and RAM used per cluster
+ifdef::cnv-cluster[]
+* Number of virtual machines in the cluster
```
Comment (pkliczewski) [inline modules/telemetry-what-information-is-collected.adoc]: We do not have this metric yet. Let's keep only virtual machine instance count.

**Comment by pkliczewski** [comment]
@aburdenthehand It looks good. Please only keep vmi count.

**Review by pkliczewski — APPROVED** [review]
Looks good to me

**Review by kalexand-rh — COMMENTED** [review]
LGTM!

**Pull request merged**